src/reducers/existingThoughtMove.js

```diff
@@ -180,11 +180,24 @@ export default (state, { oldPath, newPath, offset }) => {
     }
   })
 
+  /**
+   * Keep cursor position on child element after moving parent
+   */
+  const generateCursorNewPath = () => {
```

Comment:
I don't think we need a function for this. You can just use local variables and expressions.